client/ayon_blender/api/lib.py

```diff
@@ -587,3 +587,48 @@ def get_blender_version():
     """
     major, minor, subversion = bpy.app.version
     return major, minor, subversion
+
+
+@contextlib.contextmanager
+def strip_container_data(containers):
+    """Remove container data during context
+    """
+    container_data = {}
+    for container in containers:
+        node = container["node"]
+        container_data[node] = dict(
+            node.get(pipeline.AVALON_PROPERTY)
+        )
+        del node[pipeline.AVALON_PROPERTY]
+    try:
+        yield
+
+    finally:
+        for key, item in container_data.items():
+            key[pipeline.AVALON_PROPERTY] = item
+
+
+@contextlib.contextmanager
+def strip_namespace(containers):
+    """Strip namespace during context
+    """
+    nodes = [
+        container["node"] for container in containers
+    ]
+    namespace_dict = {}
+    for node in nodes:
+        for children in node.children_recursive:
+            original_name = children.name
+            namespace, name = original_name.rsplit(':', 1)
+            children.name = name
+            namespace_dict[children] = namespace
+
+    try:
+        yield
+
+    finally:
+        for node in nodes:
+            for children in node.children_recursive:
+                namespace = namespace_dict[children]
+                name = children.name
+                children.name = f"{namespace}:{name}"
```

Comment:
1. This would fail if a node has no `:` in the name.
2. Finally iteration can be simplified to go over the `namespace_dict`
3. `children` variable is confusing, because it refers to a single node. Should be `child`.

```suggestion
    original_namespaces = {}
    for node in nodes:
        for child in node.children_recursive:
            original_name = child.name
            if ":" not in original_name:
                continue
            namespace, name = original_name.rsplit(':', 1)
            child.name = name
            original_namespaces[child] = namespace

    try:
        yield
    finally:
        for node, original_namespace in original_namespaces.items():
           node.name = f"{namespace}:{name}"
```